src/streams.h

```diff
@@ -733,18 +733,18 @@ class CBufferedFile
         if (readNow == 0)
             return false;
         size_t nBytes = fread((void*)&vchBuf[pos], 1, readNow, src);
-        if (nBytes == 0) {
+        if (nBytes == 0)
```

Comment:
Please keep the `{}`, according to the [coding style](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c): 
"If an if only has a single-statement then-clause, it can appear on the same line as the if, without braces. In every other case, braces are required, and the then and else clauses must appear correctly indented on a new line."